target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -194,6 +194,14 @@ define Device/ubnt_nanobeam-ac
 endef
 TARGET_DEVICES += ubnt_nanobeam-ac
 
+define Device/ubnt_nanobeam-ac-gen2-wa
+  $(Device/ubnt-wa)
+  DEVICE_MODEL := NanoBeam AC
+  DEVICE_VARIANT := Gen2
```

Comment:
If we have a variant here, it would be helpful to also add a device variant to "gen1" ...

Reply:
Should I rename `ubnt_nanobeam-ac-gen1-wa`? This will "break" sysupgrade for users since they have to force. :/

Reply:
I was mainly referring to DEVICE_VARIANT variable for now, as that one is just informational and won't break anything.

I would rather not change compatible/image name at this point. Is the wa suffix needed for this device? Since we don't have it for the other ar9342 ubnt wa devices so far, I'd rather stay consistent to those unless there is a non-wa nanobeam ac...

Reply:
BTW the sysupgrade couldn't be healed by simply adding the old name to supported devices:

SUPPORTED_DEVICES += ubnt,nanobeam-ac